src/tray.cpp

```diff
@@ -348,7 +348,7 @@ bool Tray::eventFilter(QObject * object, QEvent * event)
 
 void Tray::onEditConnectionsTriggered()
 {
-    const QStringList connections_editor = QSettings{}.value(CONNECTIONS_EDITOR, QStringList{{"xterm", "-e", "nmtui-edit"}}).toStringList();
+    const QStringList connections_editor = QSettings{}.value(CONNECTIONS_EDITOR, QStringList{{"nm-connection-editor"}}).toStringList();
```

Comment:
This is the last fallback default value. In this regard if the default value would be `nm-connection-manager` what would be the benefit of the whole `nm-tray`? Then you can use the gnome's `nm-applet` directly.

You can edit the editor's executable in configuration file of `nm-tray` though. There is no need to change the fallback.

Reply:
I can't get nm-applet applet to work on wayland and the nm-tray does the basics just fine. I just find nmtui extremely limiting.

Where is that config file?

Reply:
> Where is that config file?

It uses the Qt's [`QSettings`](https://doc.qt.io/qt-5/qsettings.html) -> ($XDG_CONFIG_HOME|$XDG_CONFIG_DIRS)/nm-tray/nm-tray.conf